src/DependencyInjection/MonologExtension.php

```diff
@@ -930,10 +930,10 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 } else {
                     $config = $handler['config'] ?: [];
                     $config['access_token'] = $handler['token'];
-                    $rollbar = new Definition('RollbarNotifier', [
+                    $rollbar = new Definition('Rollbar\RollbarLogger', [
```

Comment:
The signature of the `RollbarHandler ` was modified in Monolog 2.0.0 by https://github.com/Seldaek/monolog/pull/1042. This class must be set conditionally depending on the Monolog version.

In MonologBundle v4, since Monolog 3 is required, we can make this change without condition.